### ISSUE TITLE
Better integrate watched progress and history into Stremio

### ISSUE THREAD

**Issue by johnmolina** [Stremio/stremio-features#600]
**Is your feature request related to a problem? Please describe.**
Currently Stremio only provides a buggy 1-way Trakt integration, a buggy 'Continue watching' implementation and for history you have to rely on (often incomplete) Trakt or (no metadata) Debrid history. 

**Describe the solution you'd like**
Since a full 2-way Trakt integration seems unlikely at this point why not at least implement some basic functions in Stremio itself so it can provide better watched progress and history most other media apps have as standard.

What would be needed?

- Allow marking movies as watched for all platforms (currently only possible with desktop client).
- Allow marking of whole seasons/series as watched for all platforms (whole seasons currently only possible with desktop client).

- Display a 'tick' icon on all the board/discover/library items that were completely watched on all platforms (currently only possible for movies that are also in library with desktop client).
- Display a new icon on all the board/discover/library items for series that were partly watched on all platforms.

- New category 'History' with watched history (movies/series with metadata) next to 'Board'/'Discover'/'Library' on all platforms.
Simpler alternative: Create a new official 'Watched History' add-on that displays 'History - Movie' and 'History - Series' with x last movies and series (not episodes like Trakt) watched on the board with metadata. So we have a basic history without Trakt or Debrid. Should be removable.

Later...

- Allow users to delete watched Progress/in Progress/Library items in their account. Completely reset an account.
- Allow users to import/export watched progress and library items with simple .csv files from web / account management.

- Implement a way for Stremio to send the imdb ids of last x watched items to an add-on so the addon can directly provide recommendations etc. (instead of relying on Stremio -> Trakt -> Couchmonkey -> Trakt List -> Trakt Addon -> Stremio)

**Describe alternatives you've considered**
Using Stremio with Trakt is a pain with bugged scrobbling and missing 2-way integration.

**Additional context**
I would not mind completely abandoning Trakt if Stremio could provide some of these functions itself.

**Comment by oathzed** [comment]
Nice ideas. Would be good to have trakt scrobbling auto refresh your continue watching without having to close and reopen the other device app you want to continue watching on.

**Comment by sharkooon** [comment]
Upvote for this change!

**Comment by ACMadeira** [comment]
Another upvote for it! 👍 And it would be great too if when manually marking the watched for the episodes there was at least a option for marking entire seasons as watched!

**Comment by jayseventwo** [comment]
Great idea. I would prefer stremio to do this rather than use Trackt.

**Comment by TRtomasz** [comment]
We have implemented 2 way trakt integration, you can trigger it from the user panel available a stremio.com

**Comment by SourceCipher** [comment]
I dont get it why stremio is relaying on this nonsense trakt. I can see they storing the data themselves as they are marking what I watched and the progress as well throughout all my platforms. I dont understand why they cant expose this info to the addons. I wanted to build an addon to integrate watched hist within the calendar as I often want to see which movies I saw when especially for the recommendations but I realized stremios addons can only supply the data sadly.
